Lombiq.Tests.UI/Extensions/ShortcutsUITestContextExtensions.cs

```diff
@@ -744,23 +744,25 @@ public static Task ExecuteJsonRecipeSiteSettingAsync<T>(this UITestContext conte
     public static Task EnableTimeShiftingAsync(this UITestContext context) => context.EnableFeatureDirectlyAsync(ShiftTime);
 
     /// <summary>
-    /// Sets the time shift to a specific value. If both <paramref name="days"/> and <paramref name="seconds"/> are
-    /// provided, then the <see cref="TimeSpan"/> values are added together.
+    /// Sets the time shift to a specific value.
     /// </summary>
-    public static Task SetShiftTimeAsync(this UITestContext context, double days = 0, double seconds = 0)
+    public static Task SetShiftTimeAsync(this UITestContext context, TimeSpan time)
     {
         context.EnsureValidOrchardCoreTenantScope();
-        return context.GoToAsync<ShiftTimeController>(controller => controller.Set(days, seconds));
+        return time.TotalDays >= 1.0
+            ? context.GoToAsync<TimeShiftController>(controller => controller.Set(time.TotalDays, 0))
+            : context.GoToAsync<TimeShiftController>(controller => controller.Set(0, time.TotalSeconds));
```

Comment:
In the generated URL any query string parameter with the default value are omitted, so this creates a neater and more human readable URL. I think the prettier result is worth the minuscule added complexity.

Reply:
OK!